presto-product-tests/bin/product-tests-suite-7-non-generic.sh

```diff
@@ -46,10 +46,9 @@ presto-product-tests/bin/run_on_docker.sh \
     -g two_hives \
     || suite_exit_code=1
 
-env HADOOP_BASE_IMAGE="not-used" TESTS_HIVE_VERSION_MAJOR="3" TESTS_HIVE_VERSION_MINOR="1" \
-    presto-product-tests/bin/run_on_docker.sh \
-    singlenode-hdp3 \
-    -g hdp3_only,storage_formats,hive_transactional \
+TESTS_HIVE_VERSION_MAJOR="3" TESTS_HIVE_VERSION_MINOR="1" presto-product-tests-launcher/target/presto-product-tests-launcher-*-executable.jar test run \
```

Comment:
could  you please `presto-product-tests-launcher/bin/run-launcher`, since hadoop-entrypoint needs to be confirmed by CI anyway?

Reply:
Rebased on master (in this branch presto-product-tests-launcher/bin/run-launcher doesn't exist)